python/cugraph/centrality/katz_centrality.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from cugraph.centrality import katz_centrality_wrapper
+
+
+def katz_centrality(G,
+                    alpha=0.1,
+                    max_iter=100,
+                    tol=1.0e-5,
+                    nstart=None,
+                    normalized=True):
+    """
+    Compute the Katz centrality for the nodes of the graph G
+
+    Parameters
+    ----------
+    graph : cuGraph.Graph
+        cuGraph graph descriptor with connectivity information. Edge weights,
```

Comment:
Should we talk about directed/undirected support?

Reply:
The algorithm should work for both types.

Reply:
Should an undirected edge be stored as a directed edge in both directions in the CSR matrix?

Reply:
That is what we do. We don't have two arrays with in and out. Only out edges. So each (u,v) appears twice. Once in the out of u and once in the out of v.

Reply:
That's consistent with what other cuGraph algo do. I don't see it in the doc though. We may want to clarify it here.

Reply:
I have mentioned the undirect/directness of the input graph.